blueocean-dashboard/src/main/js/util/logDisplayHelper.js

```diff
@@ -31,6 +31,9 @@ export const getNodesInformation = (nodes) => {
         const isFailingNode = errorNodes.indexOf(item.id) > -1;
         const isRunningNode = runningNodes.indexOf(item.id) > -1;
 
+        const hasLogs = item.actions ? item.actions
+                .filter(action => action._class === 'org.jenkinsci.plugins.workflow.support.actions.LogActionImpl').length > 0
```

Comment:
Should this be using capabilities cc @vivek @cliffmeyers ? (if so, can you suggest what ones to use?). 
In theory we should be looking up the capabilities list from the _class vs doing === on it directly (assuming there is a log capability).